src/scene/vcFolder.cpp

```diff
@@ -231,7 +231,20 @@ void vcFolder::HandleImGui(vcState *pProgramState, size_t *pItemID)
       bool sceneExplorerItemClicked = ((ImGui::IsMouseReleased(0) && ImGui::IsItemHovered() && !ImGui::IsItemActive()) || (!pSceneItem->m_selected && ImGui::IsItemActive()));
       if (sceneExplorerItemClicked)
       {
-        udStrcpy(pProgramState->sceneExplorer.selectUUIDWhenPossible, pNode->UUID);
```

Comment:
What ramifications are there to not using this system? Why was it used in the first place?

Reply:
I think it just seemed like a nice way to consolidate the code into a single place, which it would be if there were a single place at this point where every node in the tree were iterated through every frame. The only ramification I see is more code, I think initially that system was implemented for the case where you wanted to select a new scene item that wasn't yet instantiated